tensorflow_addons/optimizers/novograd.py

```diff
@@ -0,0 +1,246 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Novograd for TensorFlow."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+from tensorflow.python.training import training_ops
+
+
+@tf.keras.utils.register_keras_serializable(package='Addons')
+class Novograd(tf.keras.optimizers.Optimizer):
```

Comment:
Shouldn't this be `NovoGrad`?

https://github.com/NVIDIA/OpenSeq2Seq/blob/master/open_seq2seq/optimizers/novograd.py#L30
https://nvidia.github.io/OpenSeq2Seq/html/optimizers.html